tests/testapp/tests/test_controller_methods.py

```diff
@@ -0,0 +1,254 @@
+import factory
+import json
+import mock
+import uuid
+
+from django.test import TestCase
+from morango.utils.controller import MorangoProfileController
+from facility_profile.models import Facility, MyUser
+from morango.models import DatabaseIDModel, DeletedModels, InstanceIDModel, RecordMaxCounter, Store
+
+
+def serialized_facility_factory(identifier):
+    serialized = {'name': "Facility {}".format(identifier),
+                  'id': identifier,
+                  '_morango_source_id': uuid.uuid4().hex,
+                  '_morango_partition': "",
+                  '_morango_dirty_bit': False}
+    return json.dumps(serialized)
```

Comment:
May be easier/more maintainable to create a Facility instance (without saving it), and then calling `serialize` on it?